packages/subgraph/src/mapping.ts

```diff
@@ -69,6 +65,13 @@ export function processPayload(
   let reader = new BytesReader(payloadBytes);
   let blockIdx = 0;
 
+  if(cache.getGlobalState().owner != payload.submitter) {
```

Comment:
Does AssemblyScript have a `String.to_lower_case()` method or something like that?

I am only asking because `0xDEADBEEF` would be different than `0xdeadbeef`, so we should be cautious about comparing strings with the same case here.

Reply:
Shouldn't be an issue because we are using `Bytes.toHexString()` for all owner/address checks here, which normalizes to lowercase strings, although we can always add another test case 😅